PIM/API/02_GetStarted.md

```diff
@@ -0,0 +1,255 @@
+-  The attributes provided are the standard Actindo attributes contained in the *PIM basic set*. If you have created your own attributes and attribute sets, the attribute names will probably differ. If desired, you can get a list of all attributes created in an attribute set via API, see [List the attributes in an attribute set](./04_RetrieveEntityData.md#list-the-attributes-in-an-attribute-set).   
+- The scope (in this case, online shop) and languages (en_US and es_ES) must also be specified accordingly. 
+
+
+## Set up the OAuth authorization
+
+The *Actindo Core1 Platform* uses the OAuth 2.0 open protocol to handle client authorization for API access. To make authenticated API requests, you need to set up the OAuth authorization first.
+
+#### Prerequisites
+
+- An app for client access is available. 
+- A user has been created.
+- The user has the appropriate rights.
+
+### Step 1: Register the client app
+
+If you want to gain access to the Core1 from any external app via API, first of all you have to register the client app in the developer portal. 
+
+*Dev Tools > Tab APP REGISTRATIONS*
+
+![App registrations](../../Assets/Screenshots/PIM/API/AppRegistrations.png "[App registrations]")
+
+1. Click the ![App registrations](../../Assets/Icons/Plus01.png "[App registrations]") button to add an app registration.
+    The *Create* view is displayed.
+
+    ![Create app registration](../../Assets/Screenshots/PIM/API/CreateAppRegistration.png "[Create app registration]")
+
+2. Enter a name for the app registration in the *Name* field. For the sake of clarity, it is recommended to use the client app name.
+
+3. Enter a valid client ID in the *Application (client) ID* field. You can freely define your client ID, although it is recommended to use a UUID generator tool, such as [UUID Generator](https://www.uuidgenerator.net/version4 "[https://www.uuidgenerator.net/version4]"), to generate a UUID (universally unique identifier).
+
+    The client ID for the Core1 must comply with the following requirements:
+    - between 4 and a 31 characters long  
+    - allowed characters include upper and lower cases letters, numbers, and underscore
+    - beginning with a letter  
+
+[comment]: <> (Wahrscheinlich auch mit einer Zahl am Anfang möglich laut Oli. Evtl. von Devs oder ImSpecs testen/bestätigen lassen und dementsprechend anpassen! UI muss verbessert werden, da Toggle darunter auch nicht funktioniert.)
+
+4. If available in your version, leave the toggle *Only authentication scopes open ID profile email. You will not be able to call any methods besides getting profile data" disabled.
+
+5. Enter an appropriate redirect URI in the *Redirect URI* field. For exemplary purposes, we are using https://oauthdebugger.com/debug. See the following screenshot for reference.
```

Comment:
Wer ist mit "wir" gemeint?
Der Links öffnet kein neues Fenster, das ist nicht schön, haben wir das immer bei externen Links? Ich habe bei BigCommerce geguckt, dort wird ein extra Fenster geöffnet. Ich sehe gerade, Du hast den Link auch nicht "nach Vorschrift" erstellt. um das zu vermeiden, kannst Du auch Code formatieren,dann ist Link nicht ausführbar ->
`https://customeraccount.dev.actindo.com/Actindo.Extensions.Actindo.Adyen.Notification.notificationAsync`

Reply:
Ich denke, das ist ein standard Verhalten der Webseite: externe Links werden in einem neuen Tab geöffnet; Doku-interne, in demselben.
Viele Links haben tatsächlich kein "Flag" (Tooltip). Ich merke es in unserer "Verbesserung"-Liste, damit wir durchgängig prüfen können.